contracts/SportMarkets/SportsAMM.sol

```diff
@@ -999,8 +1007,10 @@ contract SportsAMM is Initializable, ProxyOwned, PausableUpgradeable, ProxyReent
         bool useBalance,
         DoubleChanceStruct memory dcs
     ) internal view returns (uint _available) {
+        (uint tag1, uint tag2) = _getTagsForMarket(market);
```

Comment:
tag2 doesnt need to be declared as its not used, so just
` (uint tag1,) = _getTagsForMarket(market);`

Reply:
done